### DISCUSSION TITLE
Release 1.4.5 ?

### DISCUSSION THREAD

**Discussion by tgodzik** [scalacenter/bloop#1391] [General]
It seems that some of the snapshot releases failed and it's not really possible to rerun them. Would it make sense to make a bugfix release 1.4.5? Especially with zipkin fix I think it's a good time to do it. @Duhemm @olafurpg  What do you think? I can take care of it.

**Comment by Duhemm** [comment]
Thanks a lot for the initiative @tgodzik! If this can wait for Tuesday, I'd like to include #1387 in the next release. I'd be happy to assist you if you need.

**Comment by tgodzik** [comment]
Ok! Let's wait then until #1387 is merged.

**Comment by tgodzik** [comment]
Just a heads up, seems that there was a race condition and when I pushed `1.4.5` tag it got picked up by the earlier CI job, which actually uploaded the 1.4.5 artifacts, while the actual release job failed. Next time we should wait for the previous CI job to finish.

I will publish the release manually for now, since I don't have access to `bloopid`, sorry about that :/

**Comment by tgodzik** [comment]
I created the release, updated brew and scoop packages. Last remaining one is the arch package, is anyone able to update it manually? Sorry for the trouble.

**Comment by tgodzik** [comment]
@TheElectronWill would you be able to update the arch package for 1.4.5 ?

**Reply by TheElectronWill** [reply]
Sure!

**Reply by TheElectronWill** [reply]
It should be updated now :)

**Reply by tgodzik** [reply]
Amazing! Thanks!
